implementations/rust/ockam/ockam_command/src/zone/repl.rs

```diff
@@ -78,9 +79,18 @@ impl ReplCommand {
             self.open_repl(&opts, address.clone(), restart_tx).await
         } else {
             let zone_config = self.zone.zone_config()?;
-            let (executors, repl_address) = self.zone_inlets(&opts, &zone_config).await?;
-            // let (repl_data, rest_inlet_handles) = cmd.dummy_inlet(&opts).await?;
+            let (executors, logs_address, repl_address) =
+                self.zone_inlets(&opts, &zone_config).await?;
+
             opts.terminal.write_line(fmt_separator!())?;
+            if let Some(logs_address) = logs_address {
+                opts.terminal.write_line(fmt_log!(
+                    "Open your browser at: {} to access the zone logs",
+                    color_primary(format!("http://localhost:{}/", logs_address.port()))
```

Comment:
I was checking the [http](https://github.com/build-trust/ockam/pull/9072/files#diff-5650b32e142d51fb1568f6631b7e76fc730556f167ac0d3c10be130db416838fR558-R559) message and they should have a similar structure so they look good when we show both.

Reply:
Yeah, makes sense to me